modules/lambdas/main.tf

```diff
@@ -146,6 +147,55 @@ resource "aws_lambda_function" "extract_filepaths_for_granule" {
 }
 
 
+data "aws_iam_policy_document" "assume_lambda_role_extract" {
+  statement {
+    principals {
+      type = "AWS"
```

Comment:
Should the type be `AWS `or `Service`?
In similar example in orca, it uses Service
https://github.com/nasa/cumulus-orca/blob/4d6cff91690984c4a8b947d89a2115d1e0009301/modules/iam/main.tf#L16